src/Angor/Avalonia/Angor.Model.Implementation/Projects/Project.cs

```diff
@@ -0,0 +1,20 @@
+using AngorApp.Model;
+
+namespace Angor.Model.Implementation.Projects;
```

Comment:
This model belongs to the UI only right? perhaps the namespace should reflect that?

Reply:
Yes, absolutely. I've modified the namespaced to reflect that.